app/components/bookings/edit/edit-label.component.html

```diff
@@ -0,0 +1,4 @@
+<div id="container">
```

Comment:
as pointed out above, elements which are rendered multiple times, should use classes instead of ids, since they won't be unique.